lib/srv/desktop/rdp/rdpclient/src/rdpdr/mod.rs

```diff
@@ -2164,6 +2496,7 @@ impl ClientDriveQueryDirectoryResponse {
         })
     }
 
+    // TODO(isaiah): make this logic more sane
```

Comment:
I'm fine with leaving a TODO, but make it something more actionable. We're not likely to remember what this even means.

Reply:
I address it a couple of PR's from this one: https://github.com/gravitational/teleport/pull/12912/files#diff-983433889acb8e2c478d0a04b26bb477bd81271f29515f1ee6c56fa881e2880aL2560